### ISSUE TITLE
502_GPU.ipynb

### ISSUE THREAD

**Issue by keineahnung2345** [MorvanZhou/PyTorch-Tutorial#59]
In the 7th cell of 502_GPU.ipynb, 
`accuracy = sum(pred_y == test_y) / test_y.size(0)` should be replaced with `accuracy = torch.sum(pred_y == test_y).type(torch.FloatTensor) / test_y.size(0)`, or it will always be 0.

**Comment by MorvanZhou** [comment]
Fixed with thanks
